source/wp-content/themes/wporg-parent-2021/patterns/404-page.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Title: 404 Page
+ * Slug: wporg-parent-2021/404-page
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"style":{"border":{"top":{"width":"1px","color":"var:preset|color|charcoal-1"},"right":{"width":"0px","style":"none"},"bottom":{"width":"0px","style":"none"},"left":{"width":"0px","style":"none"}},"spacing":{"padding":{"left":"var:preset|spacing|60","right":"var:preset|spacing|60"},"margin":{"top":"0"}}},"backgroundColor":"charcoal-2","textColor":"white","className":"site-content-container"} -->
+<main class="wp-block-group site-content-container has-white-color has-charcoal-2-background-color has-text-color has-background" style="border-top-color:var(--wp--preset--color--charcoal-1);border-top-width:1px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-style:none;border-left-width:0px;margin-top:0;padding-right:var(--wp--preset--spacing--60);padding-left:var(--wp--preset--spacing--60)"><!-- wp:group {"textColor":"charcoal-1","className":"wporg-parent-oops-container","layout":{"type":"constrained"}} -->
+<div class="wp-block-group wporg-parent-oops-container has-charcoal-1-color has-text-color"><!-- wp:paragraph {"textColor":"white"} -->
+<p class="has-white-color has-text-color">Change the text color of the group here to adjust "Oops" color.</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group -->
+
+<!-- wp:heading {"level":1} -->
+<h1><?php esc_html_e( 'This page doesn\'t exist.', 'wporg' ); ?></h1>
```

Comment:
This should be using a curly quote since it's in human-facing text.

News:
<img width="400" alt="Screenshot 2022-11-29 at 11 42 16 AM" src="https://user-images.githubusercontent.com/541093/204591702-6915c0b9-ffae-46e1-90cf-ea6393a027f1.png">

This PR:
<img width="400" alt="Screenshot 2022-11-29 at 11 42 12 AM" src="https://user-images.githubusercontent.com/541093/204591671-13a31980-ec1d-4246-941a-214bf8b25dc2.png">

Can be fixed with this:
```suggestion
<h1><?php esc_html_e( 'This page doesn&#146;t exist.', 'wporg' ); ?></h1>
```